src/components/editor/EditorArticleController.jsx

```diff
@@ -368,18 +418,18 @@ the save changes button is supposed to be disabled in this case");
       const newAuthorsAdded = update(this.state.authorsAdded, {$splice: [[index, 1]]});
       this.safeSetState({
         authorsAdded: newAuthorsAdded,
+      }, () => {
+        console.log(this.state.authorsAdded);
       });
     }
   }
 
-  handleTeaserChanges(e) {
-    let teaser = e.target.value;
+  handleTeaserChanges() {
```

Comment:
Correct me if I'm wrong, but from what I understood, this function was here to make sure the length of the teaser did not go over X number of characters. The way I refactored it to work was that the `teaser` state is what is rendered in the teaser textbox. It's not just a regular form value. By switching it to work on state, we can simply monitor the state via the `handleTeaserChanges()` function which trims the value of `teaser` when its length surpasses X. Unless there was some other functionality that I missed, the trimming functionality works like before.